app/web/server.rb

```diff
@@ -46,28 +46,24 @@ def json_params
 
     post '/todo/:todo_id' do
       command = Commands::Todo::Add::Command.new(json_params)
```

Comment:
Hmmm, yeah could do. Do you prefer that over calling validate inside the handler?